src/wattsi.pas

```diff
@@ -1848,6 +1848,39 @@ procedure Save(const Document: TDocument; const FileName: AnsiString; const InSp
          Write(F, HTMLFragment);
    end;
 
+   procedure InsertWPTTestsBlock(const Element: TElement);
+   var
+      WPTPaths, WPTOutput: TStrings;
+      WPTPath, WPTSubPath, WPTFilename: String;
+      WPTPathPrefix: String = '/html/';
+   begin
+      if (InSplit) then
+         exit;
+      if (Element.HasAttribute('pathprefix')) then
+         WPTPathPrefix := Trim(Element.GetAttribute('pathprefix').AsString);
+      WPTOutput := TStringList.Create;
+      WPTOutput.Add('<ul class=wpt-tests-block>');
+      WPTPaths := TStringList.Create;
+      WPTPaths.Text := Element.TextContent.AsString;
+      for WPTSubPath in WPTPaths do
+      begin
+         if (Trim(WPTSubPath) = '') then
+            continue;
+         WPTPath := WPTPathPrefix + Trim(WPTSubPath);
+         WPTFilename := ExtractFileName(WPTPath);
+         WPTOutput.Add('<li class=wpt-test>');
+         WPTOutput.Add('<a href="https://wpt.fyi/results'
+            + WPTPath + '">' + WPTFilename + '</a>');
+         WPTOutput.Add('<a href="http://web-platform-tests.live'
```

Comment:
This should have similar logic as

https://github.com/web-platform-tests/wpt.fyi/blob/master/webapp/components/test-runs.html#L74

for http vs https.